contracts/BridgeCommittee.sol

```diff
@@ -2,7 +2,6 @@
 pragma solidity ^0.8.20;
 
 import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
-import "./interfaces/IBridgeCommittee.sol";
```

Comment:
We still need this interface too.

Reply:
more or less the same as the others, it is inherited from `import "./utils/CommitteeUpgradeable.sol";`